compiler/rustc_mir_transform/src/lower_intrinsics.rs

```diff
@@ -107,9 +107,29 @@ impl<'tcx> MirPass<'tcx> for LowerIntrinsics {
                         }
                     }
                     sym::add_with_overflow | sym::sub_with_overflow | sym::mul_with_overflow => {
```

Comment:
If those intrinsics are lowered now, can you remove their implementation from code generation backends and interpreter.